### PR TITLE
How do I make my event private?

### PR DESCRIPTION
**Pull request by NishithP2004**
Hello I have successfully created a support page on how do I make my event private?

### CODE REVIEW

**Comment by NishithP2004** [comment]
Please check it as soon as possible.

**Comment by Masquerade0097** [comment]
Duplicate #22

File: event-setup/How-do-I-make-my-event-private.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# How do I make my event private
+Go to create event page and fill in the required details
+![img_3268](https://user-images.githubusercontent.com/34577844/48673868-0fddf580-eb6c-11e8-8141-007d2bcfe72e.jpeg)
```
Comment (Masquerade0097) [inline event-setup/How-do-I-make-my-event-private.md]: DONT use githubusercontent URL for your images. Put them in directory `event-setup/images` and use that path to load them.

Comment (abishekvashok) [reply event-setup/How-do-I-make-my-event-private.md]: Pls fix this

Comment (Masquerade0097) [reply event-setup/How-do-I-make-my-event-private.md]: @NishithP2004 See how images being loaded in other `.md` files in `event-setup` directory.

**Comment by NishithP2004** [comment]
Hello sir, I have fixed the issue and done as requested.

**Comment by NishithP2004** [comment]
@Masquerade0097

File: event-setup/How-do-I-make-my-event-private.md (hunk 1 above)
Comment (NishithP2004) [reply event-setup/How-do-I-make-my-event-private.md]: I have successfully addressed this issue @Masquerade0097

**Comment by Masquerade0097** [comment]
@mariobehling Can you please review and merge this ?

**Comment by NishithP2004** [comment]
@tabesin could you please merge this.

**Comment by tabesin** [comment]
I cannot merge but approve it

**Pull request merged**